src/Mint/Common.hs

```diff
@@ -305,41 +328,43 @@ pRemove common vrange discConfig outs sigs = plam $ \pkToRemove node -> P.do
 
   passert "signed by user." (pelem # pkToRemove # sigs)
 
-  configF <- pletFields @'["discoveryDeadline", "penaltyAddress"] discConfig
+  configF <- pletFields @'["stakingDeadline", "penaltyAddress"] discConfig
 
   let ownInputLovelace = plovelaceValueOf # removedValue
       ownInputFee = pdivideCeil # ownInputLovelace # 4
-      discDeadline = configF.discoveryDeadline
+      discDeadline = configF.stakingDeadline
 
   let finalCheck =
         -- user committing before deadline
-        pif
-          (pafter # (discDeadline - 86_400_000) # vrange) -- user committing before 24 hours before deadline
-          (pconstant True)
-          ( pany
-              # plam
-                ( \out ->
-                    pfield @"address"
-                      # out
-                      #== configF.penaltyAddress
-                      #&& ownInputFee
-                      #<= plovelaceValueOf
-                      # (pfield @"value" # out)
-                )
-              # outs -- must pay 25% fee
-          )
+        ( pif
+            (pafter # (discDeadline - 86_400_000) # vrange) -- user committing before 24 hours before deadline
```

Comment:
Just a quick info: we can actually use `24 * 60 * 60 * 1000` here, it will be calculated during the `compile` phase and the final value will be directly included in the contract (as long as we do not use the Plutarch calculation functions like `#*`).